examples/tv-app/android/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Android build folder
+android/App/app/build/
```

Comment:
There should not be anything in this folder anymore. Is there something here?

Reply:
Updated. Added to a few lines to the App/.gitignore file.